cli_scripts.py

```diff
@@ -0,0 +1,45 @@
+import subprocess
+
+
+def uvx(*cmds: str):
```

Comment:
@o-santi there is something about calling this function `uvx` that could lead to confusion since `uv` does have a `uvx` itself.